test/Raven/Tests/ClientTest.php

```diff
@@ -1281,6 +1281,36 @@ public function testSanitizeContexts()
         )), $data);
     }
 
+    /**
+     * @covers Raven_Client::sanitize
+     */
+    public function testSanitizeBreadcrumbs()
+    {
+        $client = new Dummy_Raven_Client();
+        $data = array('breadcrumbs' => array(array(
+            'category' => 'foo',
+            'message' => 'bar',
+            'data' => array(
+                'line' => 1216,
+                'bindings' => array(
+                    array('foo', 'bar'),
+                )
+            ),
+        )));
+        $client->sanitize($data);
+
+        $this->assertEquals(array('breadcrumbs' => array(array(
+            'category' => 'foo',
+            'message' => 'bar',
+            'data' => array(
+                'line' => 1216,
+                'bindings' => array(
+                    array('foo', 'bar'),
```

Comment:
It would be ideal if these tests also tested to see if `mb_convert_encoding($value, 'UTF-8');` is working properly. I'm not sure how to include binary data in the test.

Reply:
Maybe this could help? https://blog-en.openalfa.com/how-to-work-with-binary-data-in-php

Also, temporarely commenting your fix should make the test fail...

Reply:
The CI tests fail if there are PR comments? Thats kinda cool. I am going to add the binary check in the test and then rebase / squish and remove these comments.